lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -93,6 +93,33 @@ def initialize(name:)
     class RequiredParameter < Parameter
     end
 
+    class Attribute < Entry
+      extend T::Sig
+
+      sig { returns(T.nilable(Entry::Namespace)) }
+      attr_reader :owner
+
+      sig { returns(T::Array[Parameter]) }
+      attr_reader :parameters
```

Comment:
Do we really need to store this? I believe it's just used for completion right? Why not move this to a method and call it when needed?

Reply:
Yeah, good point. No need to waste memory on every attribute.